google/cloud/forseti/scanner/scanners/config_validator_util/validator_client.py

```diff
@@ -54,44 +54,7 @@ def __init__(self, endpoint=DEFAULT_ENDPOINT):
             ('grpc.max_receive_message_length', self.max_length)])
         self.stub = validator_pb2_grpc.ValidatorStub(self.channel)
 
-    @retry(retry_on_exception=retryable_exceptions.is_retryable_exception_cv,
```

Comment:
This code is not used anymore by Forseti and these API methods are no longer supported by CV.